oauth2-server/src/main/java/com/clouway/oauth2/IssueNewTokenActivity.java

```diff
@@ -64,7 +64,12 @@ public Response execute(Client client, Identity identity, Set<String> scopes, Re
               .setSubject(identity.id())
               .setIssuedAt(instant.asDate())
               .setExpiration(new Date(accessToken.expirationTimestamp()))
-              .claim("identity", identityAsJson(identity).toString())
+              .claim("id", identity.id())
+              .claim("name", identity.name())
+              .claim("email", identity.email())
+              .claim("given_name", identity.givenName())
+              .claim("family_name", identity.familyName())
+              .claim("_private_claims", privateClaims(identity).toString())
```

Comment:
Don't add them as additional property as the apps need to ensure that they are unique.  It is easier for parsing from the client side if they are at the same level as standard claims.